terralab/commands/pipeline_runs_commands.py

```diff
@@ -2,16 +2,17 @@
 
 import click
 import logging
+import uuid
 
 from terralab.logic import pipeline_runs_logic
-from terralab.utils import handle_api_exceptions, process_json_to_dict
+from terralab.utils import handle_api_exceptions, process_json_to_dict, validate_job_id
 from terralab.logic import pipelines_logic
 
 LOGGER = logging.getLogger(__name__)
 
 
-@click.command()
-@click.argument("pipeline_name")
+@click.command(short_help="Submit a pipeline run")
```

Comment:
possibly, it might make things clearer for developers. the way the help text works is that the top-level help output (what you see when you run `terralab -h`) comes from the docstrings, but if the docstrings are long or multiline, it gets cut off in sometimes arbitrary places with `...` at the end. so adding the short_help here lets you enforce what shows up there, and the longer docstring is used for the command-specific `-h` output. so I think I'd only added it in places where the top-level help output was getting truncated weirdly, but we could just add it everywhere if we prefer. what do you think?